modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/schema/yup.ts

```diff
@@ -20,10 +20,15 @@ import i18n from '../i18n';
 const MINIMUM_CHARACTERS_PASSWORD = 8;
 
 const userSchema = yup.object({
-	alternateName: yup.string().required(),
-	emailAddress: yup.string().email().required(),
-	familyName: yup.string().required(),
-	givenName: yup.string().required(),
+	alternateName: yup
+		.string()
+		.required(i18n.translate('this-field-is-required')),
+	emailAddress: yup
+		.string()
+		.email()
+		.required(i18n.translate('this-field-is-required')),
+	familyName: yup.string().required(i18n.translate('this-field-is-required')),
+	givenName: yup.string().required(i18n.translate('this-field-is-required')),
```

Comment:
Create a Constant for this Required Text, before line 20

```const DEFAULT_REQUIRED_TEXT = i18n.translate('this-field-is-required');```

Put the same **CONSTANT** in other places that may use ```i18n.translate('this-field-is-required');``` 

```suggestion
	alternateName: yup
		.string()
		.required(DEFAULT_REQUIRED_TEXT),
	emailAddress: yup
		.string()
		.email()
		.required(DEFAULT_REQUIRED_TEXT),
	familyName: yup.string().required(DEFAULT_REQUIRED_TEXT),
	givenName: yup.string().required(DEFAULT_REQUIRED_TEXT),
```